test/sagemaker_tests/pytorch/training/integration/sagemaker/__init__.py

```diff
@@ -83,7 +83,7 @@ def invoke_pytorch_estimator(
                     training_input = upload_s3_data(pytorch, **upload_s3_data_args)
                     inputs = {'training': training_input}
 
-                pytorch.fit(inputs=inputs, job_name=job_name)
+                pytorch.fit(inputs=inputs, job_name=job_name, logs=True)
```

Comment:
do we need this configuration or was it added for test purposes?

Reply:
It's just for the test purposes, so that we can see the actual training error.